orangecontrib/text/widgets/owscoredocuments.py

```diff
@@ -183,12 +210,20 @@ def callback(i: float) -> None:
         state.set_partial_result((sm, aggregation, scs))
 
 
+class SelectionMethods:
+    NONE, ALL, MANUAL, N_BEST = range(4)
+    ITEMS = "None", "All", "Manual", "Top words"
```

Comment:
Considering the widget scores the documents, `Top words` should be renamed to `Top documents`.